### DISCUSSION TITLE
posts/zimaboard-projects/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#278] [Comments]
# posts/zimaboard-projects/

I’ve had a ton of fun setting up and configuring a ZimaBoard and CasaOS over the last few weeks! While CasaOS is a great fit for your Home Server projects, I also decided to walk through over 20 other home server projects you can start today. These projects are for everyone, from the beginner, to the tinkerer, to the hardcore enthusiast! Thanks to ZimaBoard for sending this device!

https://technotim.live/posts/zimaboard-projects/

**Comment by ghost** [comment]
I love it.

**Comment by tobegram** [comment]
Hi Tim, this is Tom :)

in the near future my Zimablade is getting to me and I am looking for an opportunity to swop my google calender activities to a casaos app. Is there something equal I can use.

Thank you for your response and the content you deliver.

Regards Tom
